lib/ansible/modules/cloud/amazon/ec2_eip.py

```diff
@@ -88,13 +114,17 @@
 
 - name: associate an elastic IP with an instance
   ec2_eip:
+    name: myeip-finance
```

Comment:
Don't you need a region?

Reply:
Yes, but that is covered in the "Notes", along with authentication details.  I can fill out the full details for each example, if that is more useful?

Reply:
Just looking at the ec2_group examples, they all have region in them.  It's nice to be able to cut and paste them if you've set up the secrets... but as long as it is consistent in all the examples then it seems fine not to have them here.